app/code/Magento/Catalog/view/frontend/templates/product/image_with_borders.phtml

```diff
@@ -13,7 +13,7 @@
         <img class="<?= /* @escapeNotVerified */ $block->getClass() ?>"
             <?= /* @escapeNotVerified */ $block->getCustomAttributes() ?>
             src="<?= /* @escapeNotVerified */ $block->getImageUrl() ?>"
-            max-width="<?= /* @escapeNotVerified */ $block->getWidth() ?>"
-            max-height="<?= /* @escapeNotVerified */ $block->getHeight() ?>"
+            width="<?= /* @escapeNotVerified */ $block->getWidth() ?>"
+            height="<?= /* @escapeNotVerified */ $block->getHeight() ?>"
```

Comment:
Did you test this changes? Looks like such attribute should be moved to CSS

Reply:
Yes,
It is working Fine.

Reply:
`width` and `max-width` are not the same, so, you cannot just remove `max-` as it will change code behavior.

Reply:
There is no attribute available like max-width and max-height in Html img tag as per standard.

I am change this max-width and max-height to width and height.

It will not effected in css as i tested.

Max-width and max-height does not use in js as i check in the code.

So my changes is working fine.
And this will not change code behaviour.

Reply:
@niravkrish we must not use height and width as they were not used before.